pkg/testutil/e2eutil/prometheus.go

```diff
@@ -509,10 +509,6 @@ func createBlock(
 				app := h.Appender(ctx)
 
 				for _, lset := range batch {
-					sort.Slice(lset, func(i, j int) bool {
-						return lset[i].Name < lset[j].Name
-					})
```

Comment:
If lset becomes a string then it seems no way to sort it anymore?

Reply:
we cannot sort them anymore no; .. though we could probably `ScratchBuilder.Assign(lset).Sort()` or something like that. The only way to end up with unsorted labels is ScratchBuilder though i think?